src/Control/Lens/Traversal.hs

```diff
@@ -779,7 +779,8 @@ instance Monoid m => Applicative (Holes t m) where
 
 -- | Traverse both parts of a 'Bitraversable' container with matching types.
 --
--- Usually that type will be a pair.
+-- Usually that type will be a pair. Use 'Control.Lens.Each.each' to traverse
```

Comment:
I'd add this after last example, repeating the example:

```haskell
-- Use 'Control.Lens.Each.each' to traverse the elements of arbitrary homogeneous tuples.
-- >>> ("hello", "world","!") ^. each
-- "helloworld!"
```

Reply:
And as we on it, I'd also add an example using `Either`, so we don't get impressions this is useful for tuples only.

Reply:
ping @yoricksijsling